Redesign

GeoffroyBERNARD opened the pull request:
+ new logo
+ flex navbar
+ style for inputs (buttons)
+ added background image for body & nav
+ added font Coiny
+ button "traiter" is faded while not useful
+ added a separation line when result appears

floo51 commented on src/App.vue:
```diff
@@ -17,13 +22,68 @@ export default {
 </script>
 
+
 <style>
```
Can you read [this](https://vue-loader.vuejs.org/guide/scoped-css.html) documentation ?

floo51 commented on src/App.vue:
```diff
@@ -17,13 +22,68 @@ export default {
+}
+
+body{
+  
```
Extra space

floo51 commented on src/App.vue:
```diff
@@ -17,13 +22,68 @@ export default {
+}
+
+.slogan{
+  font-style:italic;
```
missing space : `font-style: italic;`
You should probably add a linter. See [prettier](https://github.com/prettier/prettier)

floo51 commented on src/components/Postiterie.vue:
```diff
@@ -73,11 +83,12 @@ export default {
            file : the image
     */
     previewImage: function(fieldName, file) {
+            document.getElementById("startTracking").classList.remove("hidden");
```
DOM direct manipulation is strongly discouraged if you're using a front-end framework like Vue, Angular or React.

GeoffroyBERNARD commented:
- Added scoped style
- Fixed space issues, standardized indent
- Removed useless css and console.log 
- Removed dom manipulation / now using vue rendering (for, bind)
